addons/website_sale/models/website.py

```diff
@@ -247,7 +247,8 @@ def _get_pl_partner_order(
 
         # no GeoIP or no pricelist for this country
         if not pricelists:
-            pricelists = pricelists.browse(website_pricelist_ids).filtered(check_pricelist)
+            pricelists = pricelists.sudo().browse(website_pricelist_ids).filtered(
```

Comment:
Noticing now, but why do you add the sudo? Is it needed to read the country groups ?